addons/website_sale/views/templates.xml

```diff
@@ -1693,7 +1693,7 @@
 
     <template id="product_price">
         <div itemprop="offers" itemscope="itemscope" itemtype="http://schema.org/Offer" t-attf-class="product_price mt-2 mb-3 {{'d-none' if combination_info['prevent_zero_price_sale'] else 'd-inline-block'}}">
-            <h3 class="css_editable_mode_hidden">
+            <div name="css_editable_mode_hidden" class="css_editable_mode_hidden h3">
```

Comment:
Maybe it is better to give meaningful names like `product_price_container`  and `product_list_price_container`  instead of using class name, wdyt?
 ```suggestion
            <div name="product_price_container" class="css_editable_mode_hidden h3">
```